dojo/google_sheet/views.py

```diff
@@ -39,6 +39,7 @@ def configure_google_sheets(request):
             else:
                 initial['Protect ' + field.name] = True
         initial['drive_folder_ID'] = system_settings.drive_folder_ID
+        initial['gamil_address'] = system_settings.email_address
```

Comment:
Shall we update the label here also?
```suggestion
        initial['email_address'] = system_settings.email_address
```

Reply:
Thanks for the catch @ayomawdb  :)